airbyte-integrations/bases/base-normalization/integration_tests/resources/naming/facebook_catalog_expected_nested_bigquery.json

```diff
@@ -0,0 +1,1134 @@
+{
+  "_airbyte_schema_test": {
+    "adcreatives_645_image_crops_ab1": "adcreatives_645_image_crops_ab1_5dc",
+    "adcreatives_645_image_crops_ab2": "adcreatives_645_image_crops_ab2_5dc",
+    "adcreatives_645_image_crops_ab3": "adcreatives_645_image_crops_ab3_5dc",
+    "adcreatives_cff_adlabels_ab1": "adcreatives_cff_adlabels_ab1_2f6",
+    "adcreatives_cff_adlabels_ab2": "adcreatives_cff_adlabels_ab2_2f6",
+    "adcreatives_cff_adlabels_ab3": "adcreatives_cff_adlabels_ab3_2f6",
+    "adcreatives_e61_template_url_spec_ab1": "adcreatives_e61_template_url_spec_ab1_c35",
```

Comment:
yes, I made one here: https://github.com/airbytehq/airbyte/blob/f7e01589839e409c649c8b60d8781b7ccd6562de/airbyte-integrations/bases/base-normalization/integration_tests/resources/test_primary_key_streams/data_input/catalog.json

It is part of the integration tests and the old/early "unit test" using the facebook or stripe catalog.json are a bit less useful because of that... should I remove them?
(we would get rid of the thousands of json lines)

Reply:
if there is duplicate, and if our integration tests do cover the same use cases, then yes I vote for removing the older tests.